.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: daily
+  - package-ecosystem: npm
+    directory: /
+    schedule:
+      interval: daily
```

Comment:
This seems like too often, but I expect you are going to do maintenance, so pick whatever works for you ;)

Reply:
Absolutely not too often, this means a whole day can go by without a fix. I think this is just and correct, I'll be doing the maintenance.